cli/run.go

```diff
@@ -99,6 +99,27 @@ func runAndExit(bitriseConfig models.BitriseDataModel, inventoryEnvironments []e
 	os.Exit(0)
 }
 
+func printWorkflowSteps(bitriseConfig models.BitriseDataModel, workflowToRunID string) {
```

Comment:
based on the [feature request](https://bitrise.uservoice.com/forums/235233-general/suggestions/11039397-add-full-stack-of-workflows-in-logs) i think we should print the workflow stack (BEFORE_WF1 -> BEFORE_WF2 -> TARGET_WF -> AFTER_WF), instead of the before and after workflows steps.